wolfcrypt/src/wc_port.c

```diff
@@ -614,8 +614,13 @@ int wolfSSL_CryptHwMutexUnLock(void) {
         (void)m;
         return 0;
     }
-
-
+#if defined(WOLFSSL_ESP32WROOM32_CRYPT)
+    int wc_LockMutex_ex(wolfSSL_Mutex *m, TickType_t xBlockTime)
```

Comment:
If we are adding a `wc_LockMutex_ex` can we setup a typedef or macro for a generic wolfTick_t? `typedef TickType_t wolfSSL_Tick` in wolf_port.h? That way the API template can be used by other platforms in the future?

Reply:
At this time, I moved the body of wc_LockMutex_ex into esp32_util.c because it seemed to be specific for esp32.